templates/data/generic_component_api_gateway_data.yaml

```diff
@@ -30,8 +35,29 @@ templates:
                   global:
                     - jobConfig_default
                     - jobConfig_postsubmit
+                    - jobConfig_generic_component
+                    - jobConfig_generic_component_api-gateway
                     - extra_refs_test-infra
                     - disable_testgrid
-                    - image_buildpack-golang-kubebuilder2
+                    - image_buildpack-golang-toolbox
+              - jobConfig:
+                  name: rel-api-gateway
+                  args:
+                    - "/home/prow/go/src/github.com/kyma-project/api-gateway"
+                    - "ci-release"
+                  branches:
+                    - "^\\d+\\.\\d+\\.\\d+(?:-.*)?$"
```

Comment:
```suggestion
                    - `^\d+\.\d+\.\d+(?:-.*)?$`
```

Reply:
`2022/09/06 10:37:40 Cannot parse data file templates/data/generic_component_api_gateway_data.yaml: yaml: line 47: found character that cannot start any token`